server/services/offender/responses/nextVisit.js

```diff
@@ -9,58 +8,30 @@ const {
   },
 } = require('../../../utils/enums');
 const { formatDateOrDefault } = require('../../../utils/date');
+const { fullNameOrDefault } = require('../../../utils/string');
 
-class NextVisit {
-  constructor(options = {}) {
-    this.startTime = options.startTime;
-    this.endTime = options.endTime;
-    this.status = options.status;
-    this.visitorName = options.visitorName;
-    this.visitType = options.visitType;
+module.exports = function NextVisit(response = []) {
+  if (!response?.length) {
```

Comment:
maybe push this check into the service and only create a NextVisit if we have one, rather than this [NullObject](https://en.wikipedia.org/wiki/Null_object_pattern)